impl/client/src/main/javascript/web/pentaho/ccc/visual/abstract.js

```diff
@@ -109,6 +109,10 @@ define([
       arrowVisible: false,
       opacity: 1
     },
+    trendDot_ibits: 0,
+    trendDot_imask: -1,
+    trendLine_ibits: 0,
+    trendLine_imask: -1,
```

Comment:
This would be enough:
```js
trendDot_ibits: 0,
trendDot_imask: "ShowsActivity",
trendLine_ibits: 0,
trendLine_imask: "ShowsActivity",
```
otherwise we're preventing display of the tooltip and of drilling-down.